README.md

```diff
@@ -53,8 +53,7 @@ These tools use the Remote Execution API to distribute builds to workers.
 
 ### Servers
 These applications implement the Remote Execution API to serve build requests
-from the clients above. These are then distributed to workers; some of these
-workers implement the Remote Worker API.
+from the clients above. Client requests are then distributed to workers.
```

Comment:
I would drop the last sentence ("Client requests are then distributed to workers." - see the comment below.